### PR TITLE
March Newsletter

### PR DESCRIPTION
**Pull request by gvonkoss**
Is on the slim side, all suggestions welcome!

### CODE REVIEW

File: _emails/newsletter-2019-03.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Origami Newsletter, March 2019
+companion_post_url: https://origami.ft.com/blog/2019/03/29/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+
+    This issue features hte new Origami website with a new design and new documentation; a highlight of the improved service that the Polyfill Service provides, and an explanation of how ECMAScript Modules broke our components.
```
Comment (JakeChampion) [inline _emails/newsletter-2019-03.html]:
```suggestion
    This issue features the new Origami website with a new design and new documentation; a highlight of the improved service that the Polyfill Service provides, and an explanation of how ECMAScript Modules broke our components.
```

File: _posts/2020-03-29-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: Origami Newsletter, March 2019
```
Comment (chee) [inline _posts/2020-03-29-newsletter.md]: the filename is march 2020!

File: _emails/newsletter-2019-03.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+{% include email/h2.html content="Top three things" %}
+
+{% capture text %}These are some of the bigger things we've worked on or released over the last month.{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+<!-- Thing 1 -->
+{% include email/h3.html content="Origami Website Launch" %}
+{% capture text %}
+
+    This month we've successfully launched the <a href="https://origami.ft.com/" target="_blank">new Origami website</a>. It is officially out of beta, and implements all of the new features of o-layout, which has unified the design across our documentation and our registry. In addition to the visual changes, this launch is packed full of new documentation and a more digestible spec. You can find more detailed information about the launch in <a href="https://origami.ft.com/blog/2019/03/11/site-update">this blog post</a>. As always, we welcome any feedback or questions you might have about the new website's content and UI.
```
Comment (JakeChampion) [inline _emails/newsletter-2019-03.html]: Should we link "new features of o-layout" to the o-layout page on the registry?

File: _posts/2020-03-29-newsletter.md (hunk 2 above)
Comment (gvonkoss) [reply _posts/2020-03-29-newsletter.md]: 🤦‍♀️

Comment (gvonkoss) [reply _posts/2020-03-29-newsletter.md]: we're ambitiously living in the future

File: _emails/newsletter-2019-03.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+<!-- Thing 2 -->
+{% include email/h3.html content="Polyfills served" %}
+{% capture text %}
+
+    Three years ago, the <a href="https://polyfill.io" target="_blank">Polyfill Service</a> served 400 million polyfills per month. Today, it serves <strong>5.6 billion</strong> every week, which is about <strong>800 million</strong> <em>every day</em> 💪.
```
Comment (JakeChampion) [inline _emails/newsletter-2019-03.html]:
```suggestion
    Three years ago, the <a href="https://polyfill.io" target="_blank">Polyfill Service</a> served 400 million polyfills per month. Today, it serves <strong>800 million</strong> <em>every day</em> 💪.
```
I don't think we need to mention the week total.

File: _emails/newsletter-2019-03.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+<!-- Thing 3 -->
+{% include email/h3.html content="ESM/CJS" %}
+{% capture text %}
+
+    Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌
```
Comment (JakeChampion) [inline _emails/newsletter-2019-03.html]:
```suggestion
    Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌.
```

Comment (JakeChampion) [reply _emails/newsletter-2019-03.html]: This should not be here, it should be under the special thanks section

File: _emails/newsletter-2019-03.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+
+<!-- Special thanks -->
+{% include email/h2.html content="Special thanks" %}
+
+{% capture text %}
+
+    Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌
```
Comment (JakeChampion) [inline _emails/newsletter-2019-03.html]:
```suggestion
    Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌.
```

File: _emails/newsletter-2019-03.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+
+<!-- Broader update -->
+{% include email/h2.html content="Broader update" %}
+
+{% capture text %}A digest list of other things that have happened over the last month:{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+{% capture text %}
+  - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is no longer a beta.
```
Comment (JakeChampion) [inline _emails/newsletter-2019-03.html]:
```suggestion
  - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is out of beta and ready for use.
```

File: _posts/2019-03-29-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+This month we've successfully launched the [new Origami website](https://origamit.ft.com). It is officially out of beta, and implements all of the new features of o-layout, which has unified the design across our documentation and our registry. 
+
+In addition to the visual changes, this launch is packed full of new documentation and a more digestible specification. You can find more detailed information about the launch in [this blog post](https://origami.ft.com/blog/2019/03/11/site-update)
+
+As always, we welcome any feedback or questions you might have about the new website's content and UI.
+
+### Polyfills served
+
+Three years ago, the [Polyfill Service](https://polyfill.io) served 400 million polyfills per month. Today, it serves **5.6 billion** every week, which is about **800 million** _every day_ 💪
```
Comment (JakeChampion) [inline _posts/2019-03-29-newsletter.md]:
```suggestion
Three years ago, the [Polyfill Service](https://polyfill.io) served 400 million polyfills per month. Today, it serves **800 million** _every day_ 💪.
```

File: _posts/2019-03-29-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+## Special thanks
+
+Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌
+
+## Broader update
+
+A digest list of other things that have happened over the last month.
+
+  - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is no longer in beta.
```
Comment (JakeChampion) [inline _posts/2019-03-29-newsletter.md]:
```suggestion
  - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is no longer in beta and is ready for use on production websites.
```

File: _posts/2019-03-29-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+We also wanted to update our tooling infrastructure to avoid relying on `babel-plugin-add-module-exports`. The intention is to get to a place where teams would not need to compile Origami components' JavaScript in order to use it.
+
+This shift caused a breaking change as it brought about client-side JavaScript errors for some projects and users. 
+
+We have since reverted those changes and are working on a different approach. If there is anything that we might have missed or any questions you might have, please get in touch with the team.
+
+## Special thanks
+
+Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌
```
Comment (JakeChampion) [inline _posts/2019-03-29-newsletter.md]:
```suggestion
Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌.
```

File: _posts/2019-03-29-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+## Broader update
+
+A digest list of other things that have happened over the last month.
+
+  - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is no longer in beta.
+  - MINOR: [o-chat v3.1.0](https://registry.origami.ft.com/components/o-chat@3.1.0) deprecates o-chat. If you have any questions about this component, please speak to the team.
+  - MINOR: 17 new faces were published to [our headshot image set](https://registry.origami.ft.com/components/headshot-images)
```
Comment (JakeChampion) [inline _posts/2019-03-29-newsletter.md]:
```suggestion
  - MINOR: 17 new faces were published to [our headshot image set](https://registry.origami.ft.com/components/headshot-images).
```

File: _posts/2019-03-29-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+## Special thanks
+
+Special thanks to the BizOps team for implementing the most recent version of <a href="https://registry.origami.ft.com/components/o-layout?brand=internal" target="_blank">o-layout</a> so quickly after its release, and flagging small bugs for us 🙌.
+
+## Broader update
+
+A digest list of other things that have happened over the last month.
+
+  - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is no longer in beta and is ready for use on production websites.
```
Comment (rowanmanning) [inline _posts/2019-03-29-newsletter.md]:
```suggestion
  - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is no longer in beta and is ready for use on production websites.
  - MAJOR: [origami-build-tools v8.0.0](https://github.com/Financial-Times/origami-build-tools#migrating-from-7xx-to-8xx) was released, dropping support for Node.js 6
  - MAJOR: o-date v3.0.0 was released, removing server-side support. [See migration guide](https://github.com/Financial-Times/o-date#migrating-from-v2-to-v3)
  - BETA: o-table v7.3.0-beta.1 was released, adding in table filtering. [See the registry for demos](https://registry.origami.ft.com/components/o-table@7.3.0-beta.1)
```

File: _posts/2019-03-29-newsletter.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+  - MINOR: [o-chat v3.1.0](https://registry.origami.ft.com/components/o-chat@3.1.0) deprecates o-chat. If you have any questions about this component, please speak to the team.
+  - MINOR: 17 new faces were published to [our headshot image set](https://registry.origami.ft.com/components/headshot-images).
```
Comment (rowanmanning) [inline _posts/2019-03-29-newsletter.md]:
```suggestion
  - MINOR: 17 new faces were published to [our headshot image set](https://registry.origami.ft.com/components/headshot-images).
    - MINOR: o-tracking v1.5.0 and v1.6.0 were released, thanks Asuka Ochi! [See the releases for more information](https://github.com/Financial-Times/o-tracking/releases)
  - MINOR/REVERT: the following components recieved minor bumps as part of the attempted ESM switch:
    - [o-assets](https://registry.origami.ft.com/components/o-assets)
    - [o-date](https://registry.origami.ft.com/components/o-date)
    - [o-element-visibility](https://registry.origami.ft.com/components/o-element-visibility)
    - [o-errors](https://registry.origami.ft.com/components/o-errors)
    - [o-expander](https://registry.origami.ft.com/components/o-expander)
    - [o-overlay](https://registry.origami.ft.com/components/o-overlay)
    - [o-share](https://registry.origami.ft.com/components/o-share)
    - [o-tabs](https://registry.origami.ft.com/components/o-tabs)
    - [o-video](https://registry.origami.ft.com/components/o-video)
    - [o-viewport](https://registry.origami.ft.com/components/o-viewport)
  - PATCH: [o-footer v6.1.1](https://registry.origami.ft.com/components/o-footer@6.1.1) was released, updating the demo date to 2019
  - PATCH: [o-footer-services v2.1.2](https://registry.origami.ft.com/components/o-footer-services@2.1.2) was released, updating the demo date to 2019
  - PATCH: [o-header-services v3.2.7](https://registry.origami.ft.com/components/o-header-services@3.2.7) was released, improving progressive font loading
  - PATCH: [o-forms v6.0.2](https://registry.origami.ft.com/components/o-forms@6.0.2) was released, fixing loading state styles
  - PATCH: [o-layout](https://registry.origami.ft.com/components/o-layout) patches v3.1.1–3.1.8 were released, fixing small bugs. [See the releases for more information](https://github.com/Financial-Times/o-layout/releases)
```

File: _emails/newsletter-2019-03.html [hunk 14]
```diff
@@ -0,0 +1,93 @@
+---
+title: Origami Newsletter, March 2019
+companion_post_url: https://origami.ft.com/blog/2019/03/29/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+
+    This issue features the new Origami website with a new design and new documentation; a highlight of the improved service that the Polyfill Service provides, and an explanation of how ECMAScript Modules broke our components.
```
Comment (notlee) [inline _emails/newsletter-2019-03.html]:
```suggestion
    This issue features the new Origami website with a new design and new documentation; a highlight of the improved service that the Polyfill Service provides; and an explanation of how ECMAScript Modules broke our components.
```

File: _emails/newsletter-2019-03.html [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+{% include email/h2.html content="Broader update" %}
+
+{% capture text %}A digest list of other things that have happened over the last month:{% endcapture %}
+{% include email/paragraph.html content=text %}
+
+{% capture text %}
+
+    - MAJOR: [o-lazy-load v1.0.0](https://registry.origami.ft.com/components/o-lazy-load@1.0.0) is no longer in beta and is ready for use on production websites.
+    - MAJOR: [origami-build-tools v8.0.0](https://github.com/Financial-Times/origami-build-tools#migrating-from-7xx-to-8xx) was released, dropping support for Node.js 6
+    - MAJOR: o-date v3.0.0 was released, removing server-side support. [See migration guide](https://github.com/Financial-Times/o-date#migrating-from-v2-to-v3)
```
Comment (notlee) [inline _emails/newsletter-2019-03.html]:
```suggestion
```

Sorry I stole this. 
>https://github.com/Financial-Times/origami/pull/74
https://github.com/Financial-Times/origami/pull/74

**Comment by JakeChampion** [comment]
Passes locally using the circleci command line tool:
image

Let's not allow the buggy CI cause this to not go out.

**Pull request merged**